src/apis/2fa.ts

```diff
@@ -17,17 +16,7 @@ const pin2FAEndpoints: any = {
   uri: '/2fa/2/pin',
 };
 
-class TwoFA {
-  http: Http;
-  username?: string;
-  password?: string;
-
-  constructor(credentials: InfobipAuth) {
-    this.http = new Http(credentials.baseUrl, credentials.authorization);
-    this.username = credentials.username;
-    this.password = credentials.password;
-  }
-
+class TwoFA extends HttpAPI {
```

Comment:
TwoFA casing is also looks strange. It's a single acronym separated as a two words. Maybe TFA, or Tfa? I understand it may be to differentiate between 2FA and 3FA but the uri 2fa may help with that ambiguity.